src/types/BeachballOptions.ts

```diff
@@ -38,6 +38,7 @@ export interface CliOptions {
   disallowDeletedChangeFiles?: boolean;
   prereleasePrefix?: string | null;
   configPath?: string;
+  commit: boolean;
```

Comment:
**suggestion:** I'd make this optional ->`commit?: boolean` ( default is true anyways ). With that you should be able to get rid of all those test changes where you've been forced to add `commit: true` by type checker

Reply:
I was on the fence with this one. Originally had it as optional, but then decided to go with a non-optional :D

Reply:
yeah, it is a little confusing - it's basically being "set" by that default argument that I mentioned, thus there is no reason having it as required. 

> thought: Not sure why this project doesn't use standard yargs for all the CLI orchestration, where this could have been handled in a "standard" way...

Reply:
> thought: Not sure why this project doesn't use standard yargs for all the CLI orchestration, where this could have been handled in a "standard" way...

You'd have to ask @kenotron to confirm but I think he switched from `yargs` to `yargs-parser` due to some issues in certain contexts related to statefulness of `yargs`? Or maybe just to be lighter weight, I forget.